hideShowPassword.js

```diff
@@ -314,10 +317,15 @@
         $.each(options.inheritStyles, $.proxy(function (index, prop) {
           options.styles[prop] = this.element.css(prop);
         }, this));
-        this.element.css(options.innerElementStyles).wrap(
-          $(options.element).addClass(options.className).css(options.styles)
-        );
+        this.element.css(options.innerElementStyles);
+        if (!options.useExistingParent) {
```

Comment:
Yeah it's good idea to use `wrapper.element` to either create a wrapper if it's string value e.g. "<div>" or just use it if it's jquery pointer to an DOM object. Let me redesign this next week. Thanks.